File: java-hangman-app/src/test/java/ru/polina_project/hangman/util/WordValidationTest.java
package ru.polina_project.hangman.util;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

public class WordValidationTest {
    @Test
    public void testDoesInputLineIsChar_ValidInput() {
        assertTrue(WordValidation.doesInputLineIsChar("a"));
        assertTrue(WordValidation.doesInputLineIsChar("Z"));
    }

    @Test
    public void testDoesInputLineIsChar_InvalidInput() {

        assertFalse(WordValidation.doesInputLineIsChar(""));
        assertFalse(WordValidation.doesInputLineIsChar("1"));
        assertFalse(WordValidation.doesInputLineIsChar("ab"));
    }
}

File: kotlin-hangman-app/src/test/kotlin/ru/polina_project/hangman/util/WordValidationTest.kt
package ru.polina_project.hangman.util

import org.junit.jupiter.api.Assertions.assertFalse
import org.junit.jupiter.api.Assertions.assertTrue
import kotlin.test.Test

class WordValidationTest {
    @Test
    fun testDoesInputLineIsChar_ValidInput() {
        assertTrue(doesInputLineIsChar("a"))
        assertTrue(doesInputLineIsChar("Z"))
    }

    @Test
    fun testDoesInputLineIsChar_InvalidInput() {

        assertFalse(doesInputLineIsChar(""))
        assertFalse(doesInputLineIsChar("1"))
        assertFalse(doesInputLineIsChar("ab"))
        assertFalse(doesInputLineIsChar(" "))
    }
}